index.html

```diff
@@ -0,0 +1,1280 @@
+    <title>Newsweek - Responsive Clone</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" rel="stylesheet">
+</head>
+<body>
+    
+    <div class="header-ctn">
+        <div class="header-top">
+            <div>
+                <div>
+                    <img class="weather-img" src="https://g.newsweek.com/img/weather/30.png" alt="Partly Cloudy">
+                    <span> 33° </span> <span class="w-city"> Bangalore, IN  </span>
+                </div>
+                <span class="date"> Mon, Mar 23, 2020 </span>
+            </div>
+            <div class="header-middle">
+                <a class="navbar-brand" href="#">
+                    <img class="logo-img" src="images/logo.svg" alt="logo">
+                </a>
+            </div>
+            <div>
+                <a class="nav-sign" href="#"> Sign in </a>
+                <a class="nav-subs" href="#"> Suscribe 	&nbsp; 	&nbsp; &gt; </a>
+            </div>
+        </div>
+        <nav class="nav header-nav">
+            <a class="nav-link" href="#"> U.S.</a>
+            <a class="nav-link" href="#"> World </a>
+            <a class="nav-link" href="#"> Business </a>
+            <a class="nav-link" href="#">Tech & Science</a>
+            <a class="nav-link" href="#"> Culture</a>
+            <a class="nav-link" href="#"> Newsgeek </a>
+            <a class="nav-link" href="#"> Sports </a>
+            <a class="nav-link" href="#"> Health </a>
+            <a class="nav-link" href="#"> The Debate </a>
+            <a class="nav-link" href="#"> Vantage </a>
+            <a class="nav-link" href="#"> Weather </a>
+            <a class="nav-link last-link" href="#">
+            Search
+            <img src="https://g.newsweek.com/www/images/icon-search-glass.svg" alt="">    
+            </a>
+        </nav>
+    </div>
+
+    <div class="fixed-top justify-content-between header2" id="h2">
+        <div class="header-a">
+            <img class="logo-img2" src="images/logo.svg" alt="logo">
+            <img class="weather-img" src="https://g.newsweek.com/img/weather/30.png" alt="Partly Cloudy">
+            <span> 33° </span> <span class="w-city"> Bangalore, IN  </span>
+        </div>
+        <div class="header-b">
+            <a class="nav-sign" href="#"> Sign in </a>
+            <a class="nav-subs" href="#"> Suscribe  <span>	&nbsp; 	&nbsp; &gt; </span> </a>
+            <a class="open-menu" href="#"> <img src="images/open-menu.png" alt="open menu"> </a>
+        </div>
```

Comment:
This navbar should be fixed when it appears. Yours have a nice effect but disappears when you scroll to the top, and that shouldn't happen.

Reply:
Could you please explain more about the scenario? like when it happens. 

In case we change the width we need to refresh once, to get the full functionality. 
yet the navbar works naive, but I find it works on most of the cases.